### PR TITLE
Improving content on Canada.ca

### PR DESCRIPTION
**Pull request by prycrane**
**Trello**

- https://trello.com/c/f3reEdEW

**French PR**

- https://github.com/canada-ca/systeme-conception/pull/256

**Branch**

- https://github.com/canada-ca/design-system/tree/improving-content-en-1

**English preview**

- https://deploy-preview-394--design-system-canada-ca.netlify.app/about/
- https://deploy-preview-394--design-system-canada-ca.netlify.app/partners/

**Design document**

- https://docs.google.com/document/d/1nKJ8hC0x5-l_7tgutGOM7KjZ1jDgsxOztuA7JIEWGew/edit#heading=h.c27giw6xp4p

### CODE REVIEW

File: partners/index.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        <p><strong>Date</strong>: 2021</p>
+        <p><strong>Partners</strong>: Agriculture and Agri-Food Canada; Immigration, Refugees and Citizenship Canada; Veterans Affairs Canada and Statistics Canada</p>
+      </div>
+      </a></div>
+  </div>
+  <div class="col-xs-12 col-sm-6 col-md-4 mrgn-bttm-lg">
+    <div class="card hght-inhrt"><a href="{{ site.url }}/partners/project-02.html" aria-label="Medical devices"><img src="images/blank-blue-lm.png" class="img-responsive" alt="">
+      <div class="card-container">
+        <h3>Medical devices</h3>
+        <p class="mrgn-tp-lg"><strong>Purpose</strong>: Make it easier for stakeholders subject to Canada’s medical device regulations find and understand regulatory requirements.</p>
```
Comment (donohuech) [inline partners/index.md]: Missing a 'to' between 'regulations' and 'find' ... change to: Make it easier for stakeholders subject to Canada's medical device regulations to find and understand regulatory requirements.

File: partners/index.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        <p><strong>Date</strong>: 2017</p>
+        <p><strong>Partner</strong>: Canada Revenue Agency</p>
+      </div>
+      </a></div>
+  </div>
+  <div class="col-xs-12 col-sm-6 col-md-4 mrgn-bttm-lg">
+    <div class="card hght-inhrt"><a href="{{ site.url }}/partners/project-10.html" aria-label="Business start-up activities"><img src="images/blank-blue-lm.png" class="img-responsive" alt="">
+      <div class="card-container">
+        <h3>Business start-up activities</h3>
+        <p class="mrgn-tp-lg"><strong>Purpose</strong>: Combine information from across departments so that Canadians easily get the whole picture on what the Government of Canada requires when starting a business</p>
```
Comment (donohuech) [inline partners/index.md]: Please add period after ....when starting a business.

File: partners/project-03.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+<p>Our objective was to make make it easier for Canadian businesses to find and understand government business opportunities and guidance.</p>
+<h2>Date</h2>
+<p>Spring/summer 2019</p>
+<h2>Partner department</h2>
+<ul>
+  <li>Public Services and Procurement Canada (PSPC)</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul class="lst-spcd">
+  <li>the federal government is the largest public buyer of goods and services in Canada, with purchases of approximately $23 billion annually</li>
```
Comment (donohuech) [inline partners/project-03.md]: Capital 't' at beginning... so "The federal...."
(makes it consistent with the capitalization on the other pages)

File: partners/project-09.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+<h2>Date</h2>
+<p>2019</p>
+<h2>Partner departments</h2>
+<ul>
+  <li>Service Canada</li>
+  <li>Employment and Social Development Canada (ESDC)</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>Thousands of Canadians parents apply for maternity and parental benefits each year</li>
```
Comment (donohuech) [inline partners/project-09.md]: Please remove 's' from 'Canadian'. So "Thousands of Canadian parents..."

File: partners/project-18.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+altLangPage: "https://conception.canada.ca/partenaires/project-18.html"
+date: 2024-01-03
+dateModified: 2024-01-03
+description: "Partner project: Make it easier for Canadians to make informed choices about vaccination. Date: Fall 2017."
+title: "Disease and immunization information"
+---
+<p>Make it easier for businesses to fulfill their Goods and Services Tax (GST), Harmonized Sales Tax (HST) and payroll requirements.</p>
```
Comment (donohuech) [inline partners/project-18.md]: This paragraph is incorrect. It should be: "Make it easier for Canadians to interpret immunization guidelines."

File: partners/project-18.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  <li>You plan to start a family soon, but want a sunny vacation first. Which of the countries below should you avoid because of the risk of Zika?</li>
+  <li>Your baby was born on the first of this month. At what age is the first set of vaccinations recommended?</li>
+  <li>It is flu season and a family member is pregnant. Find out whether or not she should get a flu shot while pregnant.</li>
+  <li>You don't want to miss work tomorrow but should stay away if you have the flu. Yesterday, all of the symptoms below hit: find out which ones are the signal that it's the flu.</li>
+  <li>You think you heard something on the news about people being sick in Italy. Is there a particular concern you should be aware of before you go with your baby?</li>
+  <li>You are planning a trip to Mexico. Which specific vaccinations are recommended before you go?</li>
+  <li>The weekly flu report says most cases are an H3N2 flu. Find out if that particular flu is in this year's vaccine in Canada to help persuade your partner to get a flu shot.</li>
+</ul>
+<h2>Research summary</h2>
+<p><a href="https://blog.canada.ca/research-summaries/diseases-research-summary.html">GST/HST and payroll</a></p>
```
Comment (donohuech) [inline partners/project-18.md]: This is incorrect. The link text should be: Diseases and immunization information" and the link should point to: https://blog.canada.ca/research-summaries/diseases-research-summary.html

File: partners/project-19.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+<p>Make it easier for Canadians to understand notifications about dangerous weather conditions.</p>
+<h2>Date</h2>
+<p>July 2017</p>
+<h2>Partner department</h2>
+<ul>
+  <li>Environment and Climate Change Canada</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>Flu cases result in nearly 12,000 hospitalizations in Canada each year</li>
```
Comment (donohuech) [inline partners/project-19.md]: This is incorrect. It should be: 

- Canadians check the weather with the Government of Canada more than they do anything else

- More than 550,000,000 visits to weather.gc.ca annually

File: partners/project-05.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+<h2>Date</h2>
+<p>Fall 2019</p>
+<h2>Partner departments</h2>
+<ul>
+  <li>Health Canada</li>
+  <li>Transport Canada</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>the federal government is the largest public buyer of goods and services in Canada, with purchases of approximately $23 billion annually</li>
```
Comment (donohuech) [inline partners/project-05.md]: This is incorrect. It should be: "There are thousands of recalls and safety alerts published every year. When Canadians can find and understand this information, they are safer."

**Review by donohuech — CHANGES_REQUESTED** [review]
@prycrane a few changes please. Looks good other than these- thank you!

File: partners/index.md (hunk 1 above)
Comment (prycrane) [reply partners/index.md]: Fixed.

File: partners/index.md (hunk 2 above)
Comment (prycrane) [reply partners/index.md]: Fixed.

File: partners/project-03.md (hunk 3 above)
Comment (prycrane) [reply partners/project-03.md]: Fixed.

File: partners/project-09.md (hunk 4 above)
Comment (prycrane) [reply partners/project-09.md]: Fixed.

File: partners/project-18.md (hunk 5 above)
Comment (prycrane) [reply partners/project-18.md]: Fixed.

File: partners/project-18.md (hunk 6 above)
Comment (prycrane) [reply partners/project-18.md]: Fixed.

File: partners/project-19.md (hunk 7 above)
Comment (prycrane) [reply partners/project-19.md]: Fixed.

File: partners/project-05.md (hunk 8 above)
Comment (prycrane) [reply partners/project-05.md]: Fixed.

**Review by donohuech — APPROVED** [review]

**Pull request merged**